libraries/rush-lib/src/cli/RushPnpmCommandLineParser.ts

```diff
@@ -331,6 +327,15 @@ export class RushPnpmCommandLineParser {
             );
             throw new AlreadyReportedError();
           }
+
+          // patch-commit internally calls installation under cwd instead of the common/temp folder
+          // It throws missing package.json error, so in this case, we need to set the dir to the common/temp folder here
+          if (!pnpmArgs.includes('--dir') && !pnpmArgs.includes('-C')) {
+            if (!(await FileSystem.existsAsync(`${process.cwd()}/${FileConstants.PackageJson}`))) {
```

Comment:
There are some repos that have a package.json in the root of a Rush repo, even though it's not considered by Rush/PNPM during install. Does this condition interfere with that scenario?

Reply:
I see that I mentioned this in my last round of reviews a couple years ago... I'm not sure where that landed in the comments though. Do we not need to care about this?